src/app/services/navigation.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from "@angular/core";
+import { Router } from "@angular/router";
+import AppValues from "../common/app.values";
+import ProjectLinks from "../project-links";
+
+@Injectable()
+export class NavigationService {
+    constructor(private router: Router) {}
+
+    public navigateToHome(params?: object): void {
+        this.router.navigate([ProjectLinks.urlHome], { queryParams: params });
+    }
+
+    public navigateToSearch(params?: object): void {
+        this.router.navigate([ProjectLinks.urlSearch], { queryParams: params });
+    }
+    public navigateToDatasetCreate(params?: object): void {
```

Comment:
Empty line between functions is missing in some cases in this file. Did you fully install the formatting pre-commit hook?